packages/cozy-harvest-lib/src/models/konnector/KonnectorJobWatcher.spec.js

```diff
@@ -1,3 +1,4 @@
+import CozyClient from 'cozy-client'
```

Comment:
SHouldn't we use `createMockClient` in a place like this?

Reply:
We could but this is not necessary here. The mock client as of now is mostly used to work with query data (so as to have query data already in the store).

For coherence I think yes, it would be better. Not doing this now though.